docs/advanced/catkin_shell_verbs.rst

```diff
@@ -0,0 +1,36 @@
+Shell support in ``catkin`` command
+===================================
+
+When you source the following file, you can use bash/zsh shell functions which provide added utility.
+
+.. code-block:: shell
+
+    . /opt/ros/indigo/etc/bash_completion.d/catkin_shell_verbs.{bash,zsh}
```

Comment:
Sorry, it is not correct ,and I think it is not installed, how can I configure setup.py to install ./scripts/catkin_shell_verbs.bash